ros2_batch_job/packaging.py

```diff
@@ -24,24 +24,19 @@
 
 
 def build_and_test_and_package(args, job):
+    overlay_pkgs = ' '.join(args.mixed_ros_overlay_pkgs)
+    skip_pkgs = overlay_pkgs + (' ros1_bridge' if not 'ros1_bridge' in overlay_pkgs else '')
+    print('skipping the following packages at first')
```

Comment:
"at first" isn't very specific. Maybe: `Skipping the following packages from underlay:`.